code/game/objects/effects/decals/posters/bs12.dm

```diff
@@ -293,3 +293,41 @@
 	icon_state="bsposter60"
 	name = "John Fleet poster"
 	desc = "A poster produced by the SCGF. It depicts Petty Officer John Fleet, a character used to boost troop morale, standing in front of the SCG flag proudly. Nobody knows if he really existed."
+
+///TeguStation posters
+
+/// Moth poster sprites are created by Azlanon and AyyRobotics
+/decl/poster/tegu_1
+	icon_state ="tposter1"
+	name = "Safety Moth - Poisoning"
```

Comment:
To future-proof, please change Moth to Alvani. We can tease them with the secret lore:tm: